cfunc/Makefile

```diff
@@ -8,13 +8,13 @@ COREIR_DIR ?= ../../coreir
 
 #COREIR_DIR = /nobackup/joeyliu/aha/coreir
 INCLUDE_DIR += $(COREIR_DIR)/include
-COREIR_LD_FLAGS = -L$(COREIR_DIR)/lib -Wl,-rpath,$(COREIR_DIR)/lib -lcoreir-commonlib -lcoreir -lcoreirsim -lcoreir-float
```

Comment:
@norabarlow  Why did you remove the other coreir lib?@

Reply:
I don't quite remember but it was what worked. Do you need them? I can see it adding them back in will make it work.

Reply:
Umm, it broke my test since I need the coreIR simulator and circuit instance from commonlib. No worries, I will add it since I have already accept the merge.